{"name": "test_01_login_error[data0]", "status": "passed", "attachments": [{"name": "log", "source": "5abd34f5-f679-4ba4-b276-e79115c388ea-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'Caseid': 1, 'module': '登录', 'title': '不输入用户名', 'username': '', 'password': '123654', 'expect': '请输入手机号'}"}], "start": 1558759844688, "stop": 1558759846701, "uuid": "7c661b29-a59e-4f6c-a717-2f837c8cdcd5", "historyId": "a1e027ef5dd8cfe3735eda6d86eb167f", "fullName": "Cases.test_login#test_01_login_error[data0]", "labels": [{"name": "tag", "value": "error"}, {"name": "parentSuite", "value": "Cases"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LAPTOP-VNEUB3U3"}, {"name": "thread", "value": "14360-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Cases.test_login"}]}